### ISSUE TITLE
EvoDAG classifier error

### ISSUE THREAD

**Issue by manzar96** [INGEOTEC/EvoMSA#59]
Hello!
I am trying to fit the EvoMSA with the default (EvoDAG) classifier however there is a bug as has no attribute fitness.

The error is the following:

> AttributeError: 'EvoDAG.function_selection.FunctionSelection' object has no attribute 'fitness'

**Comment by mgraffg** [comment]
Dear user,

Many thanks for finding this bug. Unfortunately, I cannot replicate it with the information given. 

Are you using the dataset that comes with EvoMSA? is it a private dataset? In case it is the former, could you provide the number of elements for each class? Is it work if you change EvoDAG with another classifier, e.g., base.EvoMSA(stacked_method="sklearn.naive_bayes.GaussianNB").

**Comment by manzar96** [comment]
Well, I am trying to use EvoDAG lib in my own project.
I followed the steps described below:
1. installed EvoDAG lib to my venv, using "pip install EvoDAG". The installed version was: EvoDAG-0.16.3
2. then I run the following code:

```from  EvoDAG.model import EvoDAGE
from sklearn.datasets import load_iris

data = load_iris()
X = data.data
y = data.target

clf = EvoDAGE(n_estimators=30, n_jobs=4).fit(X, y)

clf.fit(X,y)
```


(This example is also proposed in your repo [here](https://github.com/mgraffg/EvoDAG).)

And I receive this error:

> AttributeError: 'EvoDAG.function_selection.FunctionSelection' object has no attribute 'fitness'

I think you can now reproduce it :) 
Please let me know if you need anything else!

**Comment by mgraffg** [comment]
Many thanks for this information. I have just replicated the bug. Unfortunately, I do not know how to fix it. However, it works if you install through conda. 

conda install -c ingeotec evodag

I will keep this bug open until I fix it. 

Many thanks!

Mario

**Comment by mgraffg** [comment]
Finally, I found the bug;  it is fixed on version 0.16.4. The bug was in pip; I missed to include a pxd file. 

Many thanks, and please do not hesitate to contact me for any other problems or comments. 

Mario.
